apps/modernization-ui/src/apps/page-builder/page/management/edit/question/GroupQuestion/group-question.module.scss

```diff
@@ -0,0 +1,30 @@
+@use 'styles/colors';
+
+.groupQuestion {
+    height: calc(100% - 3.5rem);
```

Comment:
Doing a calc for height seems like it could cause issues when resizing the browser. 
**No validation of this concern has been performed.** :)